### PR TITLE
Skeleton for JRE 8

### PR DESCRIPTION
**Pull request by supl**
This PR creates a Dockerfile for the internal JRE 8 Docker image remediation.
It also provides GitHub Actions workflows for the scan (by push events and periodically) and release.

### CODE REVIEW

**Comment by supl** [comment]
@brfrn169 
Would be good to have your opinion.

Currently, many projects use `openjdk:8u292-jre-slim` as the base image and patch code in the Dockerfile for CVE vulnerabilities, like,
```
FROM openjdk:8u292-jre-slim

# Fix CVE-2021-3520, CVE-2021-33560, CVE-2021-20231, CVE-2021-20232, CVE-2020-24659, CVE-2021-20305, and CVE-2021-3711
RUN apt-get update && \
    apt-get install -y --no-install-recommends liblz4-1=1.8.3-1+deb10u1 libgcrypt20=1.8.4-5+deb10u1 libgnutls30=3.6.7-4+deb10u7 libhogweed4=3.4.1-1+deb10u1 libssl1.1=1.1.1d-0+deb10u7 openssl=1.1.1d-0+deb10u7 && \
    rm -rf /var/lib/apt/lists/*
```

This repository tries to maintain Dockerfiles to build internal docker images (now, only JRE 8) that can be used by the other projects to solve the annoys.

The internal image will be named `ghcr.io/scalar-labs/8jre`.

The first image uses `openjdk/8u292-jre-slim` as the base image and will be tagged as `1.0.0`.

Every time a new CVE patch is updated to the Dockerfile, we will increase the bugfix version, like `1.0.1`.

Do you have other recommendations for how to tag the image?
For example, if we update the base (upstream) image's JRE version to `openjdk/8u311-jre-slim`, and we can tag our image as `1.1.0`.

**Comment by brfrn169** [comment]
@supl Thank you for raising this PR! It's very useful.

> Every time a new CVE patch is updated to the Dockerfile, we will increase the bugfix version, like 1.0.1.
> 
> Do you have other recommendations for how to tag the image?
> For example, if we update the base (upstream) image's JRE version to openjdk/8u311-jre-slim, and we can tag our image as 1.1.0.

It sounds good to me. Thanks.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: .github/workflows/8jre-release.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Check out codes
+        uses: actions/checkout@v2
+
+      - name: Build Docker image
+        run: docker build -t ghcr.io/scalar-labs/8jre:${{ steps.version.outputs.version }} ./jre/8
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: ghcr.io/scalar-labs/8jre:${{ steps.version.outputs.version }}
```
Comment (feeblefakie) [inline .github/workflows/8jre-release.yml]:
```suggestion
          image-ref: ghcr.io/scalar-labs/jre8:${{ steps.version.outputs.version }}
```
I think `<runtime><version>` is easier to manage but is there any particular reason for the naming?
(it is also inconsistent with the directory structure (jre/8))

**Review by feeblefakie — COMMENTED** [review]
Thank you!
Left one question/suggestion.

File: .github/workflows/8jre-release.yml (hunk 1 above)
Comment (supl) [reply .github/workflows/8jre-release.yml]: @feeblefakie 
Understood.
Let me revise the naming.
(There's no particular reason for 8jre. Sorry for my bad naming... )

Comment (supl) [reply .github/workflows/8jre-release.yml]: @feeblefakie 
I changed the naming in https://github.com/scalar-labs/docker/pull/1/commits/7be024e51df7aa6060f9e598db9bd0c35cd82db3
Please take a look when you have a chance.
Thank you.

If it's possible, could you create a new secret `CR_PAT` of your personal access token?
Thank you :D

Comment (feeblefakie) [reply .github/workflows/8jre-release.yml]: I think CR_PAT is configured organization-wide, so should be fine.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**